apps/backend/src/modules/groups/group.controller.ts

```diff
@@ -0,0 +1,77 @@
+import { APIPath } from "~/libs/enums/enums.js";
+import {
+	type APIHandlerOptions,
+	type APIHandlerResponse,
+	BaseController,
+} from "~/libs/modules/controller/controller.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { type Logger } from "~/libs/modules/logger/logger.js";
+import { groupCreateValidationSchema } from "~/modules/groups/groups.js";
+
+import { type GroupService } from "./group.service.js";
+import { type GroupCreateRequestDto } from "./libs/types/types.js";
+
+class GroupController extends BaseController {
+	private groupService: GroupService;
+
+	public constructor(logger: Logger, groupService: GroupService) {
+		super(logger, APIPath.GROUPS);
+
+		this.groupService = groupService;
+
+		this.addRoute({
+			handler: (options) =>
+				this.createGroup(
+					options as APIHandlerOptions<{ body: GroupCreateRequestDto }>,
+				),
+			method: "POST",
+			path: "/create",
```

Comment:
```suggestion
			path: "/",
```
should be just a root